diff --git a/node_modules/@react-native-community/cli/build/index.js b/node_modules/@react-native-community/cli/build/index.js
index 83e9a82..3418aef 100644
--- a/node_modules/@react-native-community/cli/build/index.js
+++ b/node_modules/@react-native-community/cli/build/index.js
@@ -149,6 +149,7 @@ function attachCommand(command, config) {
   }
   cmd.addHelpText('after', printExamples(command.examples));
   for (const opt of command.options || []) {
+    if (opt.name === '--verbose') continue;
     cmd.option(opt.name, opt.description ?? '', opt.parse || (val => val), typeof opt.default === 'function' ? opt.default(config) : opt.default);
   }
 }
